<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComponentParentPageSelector</class>
 <widget class="QWidget" name="ComponentParentPageSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>8</number>
   </property>
   <property name="verticalSpacing">
    <number>4</number>
   </property>
   <item row="5" column="1">
    <widget class="QComboBox" name="comboRight"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelMiddle">
     <property name="text">
      <string>Middle:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelRight">
     <property name="text">
      <string>Right:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelLeft">
     <property name="text">
      <string>Left:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="comboMiddleRight"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelMiddleLeft">
     <property name="text">
      <string>Middle Left:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelMiddleRight">
     <property name="text">
      <string>Middle Right:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboLeft"/>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboMiddleLeft"/>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="comboMiddle"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelParentPage">
     <property name="text">
      <string>Parent Page:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboParentPage"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
